lib/content-services/src/lib/document-list/components/library-name-column/library-name-column.component.ts

```diff
@@ -62,36 +58,29 @@ export class LibraryNameColumnComponent implements OnInit, OnDestroy {
 
     private onDestroy$ = new Subject<boolean>();
 
-    constructor(
-        private element: ElementRef,
-        private nodesApiService: NodesApiService
-    ) {}
+    constructor(private element: ElementRef, private nodesApiService: NodesApiService) {}
 
     ngOnInit() {
         this.updateValue();
 
-        this.nodesApiService.nodeUpdated
-            .pipe(takeUntil(this.onDestroy$))
-            .subscribe(node => {
-                const row: ShareDataRow = this.context.row;
-                if (row) {
-                    const { entry } = row.node;
-
-                    if (entry === node) {
-                        row.node = { entry };
-                        this.updateValue();
-                    }
+        this.nodesApiService.nodeUpdated.pipe(takeUntil(this.onDestroy$)).subscribe((node) => {
+            const row: ShareDataRow = this.context.row;
+            if (row) {
+                const { entry } = row.node;
+
+                if (entry === node) {
+                    row.node = { entry };
+                    this.updateValue();
                 }
-            });
+            }
+        });
     }
 
     protected updateValue() {
         this.node = this.context.row.node;
         const rows: Array<ShareDataRow> = this.context.data.rows || [];
-        if (this.node && this.node.entry) {
-            this.displayText$.next(
-                this.makeLibraryTitle(this.node.entry as any, rows)
-            );
+        if (this.node?.entry) {
+            this.displayText$.next(this.makeLibraryTitle(this.node.entry as any, rows));
```

Comment:
Maybe we can define the type in the `makeLibraryTitle` so we can avoid `as any` conversion?

Reply:
seems like some hack, as the function accepts `library: Site` and converting to Site is not easy... feels like old code smell

Reply:
Maybe we can raise a Jira to deal with those as well? Not to block your PR

Reply:
created https://alfresco.atlassian.net/browse/ACS-5995